### DISCUSSION TITLE
can't find heading

### DISCUSSION THREAD

**Discussion by KnowProgrammer** [jgchristopher/obsidian-clipper#67] [General]
I tried using the clipper and it doesn't work. I followed the documentation and video.

I keep getting : can't find header

See the video:

https://komododecks.com/recordings/9nnoqwkIlfmCzM3FYA8F

**Comment by jgchristopher** [comment]
Your video doesn't show what your `Daily Note` template looks like. Does your daily note get created with a `# Clippings` section?
